src/Java/IO/JPEGWriter.java

```diff
@@ -0,0 +1,78 @@
+import vtk.vtkImageCanvasSource2D;
+import vtk.vtkJPEGWriter;
+import vtk.vtkNamedColors;
+import vtk.vtkNativeLibrary;
+
+public class JPEGWriter 
+{
+
+  //-----------------------------------------------------------------
+  // Load VTK library and print which library was not properly loaded
+  static 
+  {
+    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+    {
+      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+      {
+        if (!lib.IsLoaded()) 
+        {
+          System.out.println(lib.GetLibraryName() + " not loaded");
+        }
+      }
+    }
+    vtkNativeLibrary.DisableOutputWindow(null);
+  }
+  // -----------------------------------------------------------------
+  
+  public static void main(String args[]) 
+  {
+	  vtkNamedColors colors = new vtkNamedColors();
+	  
+	  double bkgColor[] = new double[] {255, 0,0,1};
+	  double boxColor[] = new double[] {255,255,0,1};
+	  
+	  colors.SetColor("bkgColor", bkgColor);
+	  colors.SetColor("boxColor", boxColor);
+	  
+	  
+	// Provide default values.
+	String file_Name = "Output.jpg";
+	String File_Argument ="";
+	for(int i = 0; i < args.length; ++i)
+	{
```

Comment:
Why do you need **File_Argument**? Use for **file_Name** use either camel case **fileName** or underscore all lower case **file_name**.